runtime/src/bank.rs

```diff
@@ -1823,17 +1860,22 @@ impl Bank {
             // ------------------+------------+-------------------+-----------------------
             // 3       20..30    | [7..8]     |    7.. 8          |    7.. 8
             //                   | [8..9]     |    8.. 9          |    8.. 9
-            // 4       30..40    | [0..0]     | <10>..10          |  <9>..10 <= not gapped
+            // 4       30..40    | [0..0]     | <10>..10          |  <9>..10 <-- not gapped
             //                   | [0..1]     |   10..11          |   10..11
             //                   | [1..2]     |   11..12          |   11..12
-            //                   | [2..9      |   12..19          |   12..19
-            // 5       40..50    |  0..4]     | <20>..24          | <19>..24 <= gapped
+            //                   | [2..9 gen=1|   12..19          |   12..19   <-+
+            // 5       40..50    |  0..0 gen=1| <20>..<20>  (noop)| <19>..<19> <-+- gapped
+            //                   |  0..4]     | <20>..24          | <19>..24   <-+
             //                   | [4..5]     |   24..25          |   24..25
             //                   | [5..6]     |   25..26          |   25..26
             // *: The range of parent_bank.slot() and current_bank.slot() is firstly
             //    split by the epoch boundaries and then the split ones are given to us.
             //    The oritinal ranges are denoted as [...]
```

Comment:
This table summarizes the changes well. :)

Reply:
"The original ranges are"